wordpress-mobile/WordPress-Android
# My Site Dashboard: Phase 1 - Quick Actions: Hide more menu

Fixes #15193

This PR hides more menu on the Quick Actions card

![more_menu](https://user-images.githubusercontent.com/1405144/130947579-e7b8ea4e-7906-4a7e-bbe0-06e7ad24d8c7.png)

**To test**

1. Launch the app
2. Navigate to My Site -> App Settings -> Test feature configuration
3. Enable the MySiteImprovementsFeatureConfig and tap the button to restart the app
4. Navigate to the My Site tab
5. Notice that the more menu is not visible on the Quick Actions (Shortcuts) card

## Regression Notes
1. Potential unintended areas of impact
More menu on the "Quick Start" card

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested manually

3. What automated tests I added (or what prevented me from doing so)
Added unit test to check that the more menu is visible on the "Quick Start" card

PR submission checklist:
